x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/flyout_container.tsx

```diff
@@ -0,0 +1,159 @@
+
+function fromExcludedClickTarget(event: Event) {
+  for (
+    let node: HTMLElement | null = event.target as HTMLElement;
+    node !== null;
+    node = node!.parentElement
+  ) {
+    if (
+      node.classList!.contains(DONT_CLOSE_DIMENSION_CONTAINER_ON_CLICK_CLASS) ||
+      node.classList!.contains('euiBody-hasPortalContent') ||
+      node.getAttribute('data-euiportal') === 'true'
+    ) {
+      return true;
+    }
+  }
+  return false;
+}
+
+export function FlyoutContainer({
+  isOpen,
+  groupLabel,
+  handleClose,
+  isFullscreen,
+  panelRef,
+  children,
+}: {
+  isOpen: boolean;
+  handleClose: () => boolean;
+  children: React.ReactElement | null;
+  groupLabel: string;
+  isFullscreen: boolean;
+  panelRef: (el: HTMLDivElement) => void;
+}) {
+  const [focusTrapIsEnabled, setFocusTrapIsEnabled] = useState(false);
+
+  const closeFlyout = useCallback(() => {
+    const canClose = handleClose();
+    if (canClose) {
+      setFocusTrapIsEnabled(false);
+    }
+    return canClose;
+  }, [handleClose]);
+
+  useEffect(() => {
+    document.body.classList.toggle('lnsBody--overflowHidden', isOpen);
+    return () => {
+      if (isOpen) {
+        setFocusTrapIsEnabled(false);
+      }
+      document.body.classList.remove('lnsBody--overflowHidden');
+    };
+  }, [isOpen]);
+
+  if (!isOpen) {
+    return null;
+  }
+
+  return (
+    <div ref={panelRef}>
+      <EuiFocusTrap
+        disabled={!focusTrapIsEnabled}
+        clickOutsideDisables={false}
+        onClickOutside={(event) => {
+          if (isFullscreen || fromExcludedClickTarget(event)) {
+            return;
+          }
+          closeFlyout();
+        }}
+        onEscapeKey={closeFlyout}
+      >
+        <div
+          role="dialog"
+          aria-labelledby="lnsDimensionContainerTitle"
+          className="lnsDimensionContainer euiFlyout"
+          onAnimationEnd={() => {
+            if (isOpen) {
+              // EuiFocusTrap interferes with animating elements with absolute position:
+              // running this onAnimationEnd, otherwise the flyout pushes content when animating
+              setFocusTrapIsEnabled(true);
+            }
+          }}
+        >
+          <EuiFlyoutHeader hasBorder className="lnsDimensionContainer__header">
+            <EuiFlexGroup gutterSize="m" alignItems="center" responsive={false}>
+              <EuiFlexItem grow={true}>
+                <EuiTitle size="xs">
+                  <h2
+                    id="lnsDimensionContainerTitle"
+                    className="lnsDimensionContainer__headerTitle"
+                  >
+                    <strong>
+                      {i18n.translate('xpack.lens.configure.configurePanelTitle', {
+                        defaultMessage: '{groupLabel}',
+                        values: {
+                          groupLabel,
+                        },
+                      })}
+                    </strong>
```

Comment:
I believe the `strong` element wrapping this text can be removed, as the same bolded style is already being applied by the parent `EuiTitle` component.

```suggestion
                    {i18n.translate('xpack.lens.configure.configurePanelTitle', {
                      defaultMessage: '{groupLabel}',
                      values: {
                        groupLabel,
                      },
                    })}
```

Reply:
Integrated feedback in latest commit